src/main/java/com/urswolfer/intellij/plugin/gerrit/GerritSettings.java

```diff
@@ -311,8 +312,11 @@ public String getCloneBaseUrl() {
         return cloneBaseUrl;
     }
 
-
     public void setLog(Logger log) {
         this.log = log;
     }
+
+    public String getCloneBaseUrlOrHost() {
+        return StringUtils.isEmpty(cloneBaseUrl) ? host : cloneBaseUrl;
```

Comment:
Use Strings.isNullOrEmpty here (this guava Strings class is already imported here).

Reply:
Done